oap-server/analyzer/meter-analyzer/src/main/java/org/apache/skywalking/oap/meter/analyzer/Analyzer.java

```diff
@@ -374,4 +374,17 @@ private void processRelationClientSide(MeterEntity entity) {
         MetricsStreamProcessor.getInstance().in(metrics);
     }
 
+    public boolean filter(ImmutableMap<String, SampleFamily> sampleFamilies) {
```

Comment:
```suggestion
    private boolean filter(ImmutableMap<String, SampleFamily> sampleFamilies) {
```
Should this be private and be placed in the later place of this file?